### DISCUSSION TITLE
Hit ESC after hitting the last note of the map to skip directly to the result screen.

### DISCUSSION THREAD

**Discussion by nenesekai** [ppy/osu#23953] [Ideas] [answered]
What it is now: After hitting the last note of a map, when you hit esc you will skip the result screen and head straight to song selection menu.

What it should be: After hitting the last note when you hit ESC it should skip the wait and go to result screen.

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/18089.
